<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mdi</class>
 <widget class="QMainWindow" name="Mdi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kitchen Sink - Qt Framework</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1298</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit_Program"/>
   </widget>
   <widget class="QMenu" name="menuLayouts">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionTableView"/>
    <addaction name="actionTreeView"/>
    <addaction name="actionListView"/>
    <addaction name="separator"/>
    <addaction name="actionTableWidget_View"/>
    <addaction name="actionTableView_Delegate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CopperSpice"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWidgets">
    <property name="title">
     <string>Widgets</string>
    </property>
    <addaction name="actionCalendar"/>
    <addaction name="actionLineEdit"/>
    <addaction name="actionSliders"/>
    <addaction name="actionTabs"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Samples</string>
    </property>
    <addaction name="actionColorPicker"/>
    <addaction name="actionFonts"/>
    <addaction name="separator"/>
    <addaction name="actionDraw"/>
    <addaction name="actionInternational"/>
    <addaction name="actionMusicPlayer"/>
    <addaction name="actionPieChart"/>
    <addaction name="actionSystemTray"/>
    <addaction name="separator"/>
    <addaction name="actionStandardDialog"/>
    <addaction name="actionStyleSheetDialog"/>
    <addaction name="separator"/>
    <addaction name="actionHTML_Viewer"/>
    <addaction name="actionWebBrowser"/>
    <addaction name="separator"/>
    <addaction name="actionXml"/>
    <addaction name="actionXmlPatterns"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuLayouts"/>
   <addaction name="menuWidgets"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit_Program">
   <property name="text">
    <string>Exit Kitchen Sink</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTreeView">
   <property name="text">
    <string>Tree View Model</string>
   </property>
  </action>
  <action name="actionListView">
   <property name="text">
    <string>List View Model</string>
   </property>
  </action>
  <action name="actionLineEdit">
   <property name="text">
    <string>LineEdit</string>
   </property>
  </action>
  <action name="actionSliders">
   <property name="text">
    <string>Sliders</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="text">
    <string>Calendar</string>
   </property>
  </action>
  <action name="actionPushButtons">
   <property name="text">
    <string>Push Buttons</string>
   </property>
  </action>
  <action name="actionStandardDialog">
   <property name="text">
    <string>Standard Dialogs</string>
   </property>
  </action>
  <action name="actionWebBrowser">
   <property name="text">
    <string>Web Browser</string>
   </property>
  </action>
  <action name="actionFonts">
   <property name="text">
    <string>Font Selector</string>
   </property>
  </action>
  <action name="actionColorPicker">
   <property name="text">
    <string>Color Selector</string>
   </property>
  </action>
  <action name="actionStyleSheetDialog">
   <property name="text">
    <string>Dialog Style Sheets</string>
   </property>
  </action>
  <action name="actionHTML_Viewer">
   <property name="text">
    <string>HTML Viewer</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Basic Drawing</string>
   </property>
  </action>
  <action name="actionTableView">
   <property name="text">
    <string>Table View Model</string>
   </property>
  </action>
  <action name="actionTableView_Delegate">
   <property name="text">
    <string>Table View Delegate</string>
   </property>
  </action>
  <action name="actionInternational">
   <property name="text">
    <string>International</string>
   </property>
  </action>
  <action name="actionMusicPlayer">
   <property name="text">
    <string>Music Player</string>
   </property>
  </action>
  <action name="actionPieChart">
   <property name="text">
    <string>Pie Chart</string>
   </property>
  </action>
  <action name="actionSystemTray">
   <property name="text">
    <string>System Tray</string>
   </property>
  </action>
  <action name="actionTableWidget_View">
   <property name="text">
    <string>Table Widget / View</string>
   </property>
  </action>
  <action name="actionAbout_CopperSpice">
   <property name="text">
    <string>About CopperSpice</string>
   </property>
  </action>
  <action name="actionXml">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionXmlPatterns">
   <property name="text">
    <string>XML Patterns</string>
   </property>
  </action>
  <action name="actionTabs">
   <property name="text">
    <string>Tabs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
